batchflow/models/tf/unet.py

```diff
@@ -59,14 +57,32 @@ class UNet(EncoderDecoder):
     def default_config(cls):
         config = super().default_config()
 
-        config['initial_block'] = dict(layout='cna cna', kernel_size=3, filters=64)
-
         config['body/encoder/num_stages'] = 4
-        config['body/encoder/blocks'] = dict(layout='cna cna', kernel_size=3, filters=[128, 256, 512, 1024])
-        config['body/embedding'] = None
-        config['body/decoder/blocks'] = dict(layout='cna cna', kernel_size=3, filters=[512, 256, 128, 64])
-
-        config['head'] = dict(layout='cna cna', kernel_size=3, filters=64)
+        config['body/encoder/order'] = ['block', 'skip', 'downsampling']
+        config['body/encoder/blocks'] = dict(layout='cna cna', kernel_size=3)
+        config['body/embedding'] = dict(layout='cna cna', kernel_size=3)
+        config['body/decoder/order'] = ['upsampling', 'combine', 'block']
+        config['body/decoder/blocks'] = dict(layout='cna cna', kernel_size=3)
 
         config['loss'] = 'ce'
         return config
+
+    def build_config(self, names=None):
+        config = super().build_config(names)
+
+        num_stages = config.get('body/encoder/num_stages')
+
+        if config.get('body/encoder/blocks/filters') is None:
+            config['body/encoder/blocks/filters'] = [64 * 2**i for i in range(num_stages)]
```

Comment:
If we want to control the number of filters in 1st encoder bock, this can be easily done. The main problem with it is how to name this parameter and where to place it in config. Any suggestions are welcome!

Reply:
Maybe, `config['common/filters_base']`?